README.md

```diff
@@ -40,5 +45,15 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, follow these instructions:
+
+* Copy the bundle (nexus-repository-apt-1.0.2.jar) into <nexus_dir>/deploy
```

Comment:
@bradbeck suggestion is interesting - although a little obfuscated in that it implies Sonatype is shipping with it by listing it inside that file. It makes me nervous...

Further I doubt we want that feature file to be mucked with - there should be a more standard  approach for adding adhoc features.